roles/cs.redis/tasks/configure.yml

```diff
@@ -111,8 +111,23 @@
     - "echo never > /sys/kernel/mm/transparent_hugepage/defrag"
   when: _redis_enabled_thd.changed
 
-- name: Ensure Redis service is started
+- name: Stop Redis/Valkey in case of migration
   service:
-    name: "{{ redis_daemon }}"
+    name: >-
+      {{ redis_daemon
+         | regex_replace('^redis', 'valkey')
+         | regex_replace('^valkey', 'redis') if redis_daemon.startswith('valkey') else
+         redis_daemon | regex_replace('^redis', 'valkey') }}
```

Comment:
maybe define `redis_uninstalling_daemon`?

```redis_daemon | regex_replace('^redis', 'valkey') | regex_replace('^valkey', 'redis')```
`redis-instance` -> `valkey-instance` -> `redis-instance`
and this branch is only active when redis_daemon starts with `valkey`
so I have no idea why there is this first regex_replace